TMessagesProj/src/main/res/values/cg_strings.xml

```diff
@@ -245,15 +244,15 @@
 
     <string name="AS_Header_Record">Media</string>
     <string name="CG_HqRoundVideos">High resolution round videos</string>
-    <string name="CG_HqRoundVideosAudio">High bitrate on round video\'s sound</string>
+    <string name="CG_HqRoundVideosAudio">Enables higher bitrate round video\'s sound</string>
     <string name="CG_StereoVoices">Record voices in stereo</string>
-    <string name="CG_StereoVoices_Desc">can be unsupported by your device!</string>
+    <string name="CG_StereoVoices_Desc">Not supported on devices with just one microphone</string>
     <string name="CG_VoiceEnhancements">Voice Enhancements</string>
-    <string name="CG_VoiceEnhancements_Desc">Noise Suppression and voice normalization</string>
+    <string name="CG_VoiceEnhancements_Desc">Enables noise suppression and voice normalization</string>
     <string name="CG_Voip_ForceEnhancements">Force VoIP Voice Enhancements</string>
-    <string name="CG_Voip_ForceEnhancements_Desc">enables AEC/NS even if server doesn\'t support it</string>
-    <string name="CG_AudioFocus">Mute music while playing voices or videos</string>
-    <string name="CG_AudioFocus_Desc">If disabled then the music will be quieter, but won\'t turn off</string>
+    <string name="CG_Voip_ForceEnhancements_Desc">Force enables AEC/NS Enhancements</string>
+    <string name="CG_AudioFocus">Mute audio from other sources while playing in-app media</string>
+    <string name="CG_AudioFocus_Desc">If disabled, temporarily turns the volume down for other media sources</string>
```

Comment:
Nice changes, but I bet that line 249 needed a change, because I think the stereo settings don't require 2 mics *(Or maybe I'm wrong, need clarification from the main developer)*

Reply:
Well, from what I know, the second microphone usually used for noise cancellation is used to achieve true stereo recording.